src/test/test_client.py

```diff
@@ -16,6 +16,12 @@ def setUp(self) -> None:
 
     def tearDown(self) -> None:
         self.client.shutdown()
+    
+    def test_amplitude_client_init_multiple_instance(self):
+        #  test multiple instance inited with different API keys
+        client1 = Amplitude(api_key="test api key 1")
+        client2 = Amplitude(api_key="test api key 2")
+        self.assertNotEqual(client1.configuration.api_key, client2.configuration.api_key)
```

Comment:
Do we need to check if the storage are different?

Reply:
Added